launch/pacmod_game_control.launch

```diff
@@ -25,7 +25,7 @@
        VEHICLE_4
        VEHICLE_5
        VEHICLE_6 -->
-  <arg name="pacmod_vehicle_type" default="LEXUS_RX_450H" />
+  <arg name="pacmod_vehicle_type" default="POLARIS_RANGER" />
```

Comment:
Please clear the changes to the launch file. Those who are using the Lexus (which has been the default for a while) will have problems if this gets changed.